engines/jsonengine.py

```diff
@@ -120,6 +118,13 @@ def table_exists(self, dbname, tablename):
         tablename = self.table_name(name=tablename, dbname=dbname)
         return os.path.exists(tablename)
 
+    def to_csv(self):
+        """Export table from json engine to CSV file"""
+        from retriever.lib.tools import json2csv, sort_csv
```

Comment:
Same as above. Imports should be at the top of the module unless there's a good reason not to have them there.